where_to_get_help.Rmd

```diff
@@ -0,0 +1,44 @@
+# Where to get help 
+
+If you are working on R it is possible that you will come across an issue where you would need some assistance to solve it. If you require help, there are options available to seek assistance or get some feedback which are discussed in this chapter. If the question involves process or tool usage then please check the rest of this guide first as it should answer your question. Please be polite, patient, and respectful of people's time while you ask for feedback or help. 

```

Comment:
Perhaps add a note here. "Please make sure to search the documentation and resources to see if your question has already been addressed. If not, please ask for assistance in the appropriate forum. Many developers are volunteers and please be thoughtful when requesting help.